.github/workflows/release.yml

```diff
@@ -153,4 +153,4 @@ jobs:
           X_ACCESS_TOKEN: ${{ secrets.X_ACCESS_TOKEN }}
           X_ACCESS_TOKEN_SECRET: ${{ secrets.X_ACCESS_TOKEN_SECRET }}
         run: |
-          VERSION=${GITHUB_REF_NAME#v} rake release:announce
+          VERSION=${GITHUB_REF_NAME#v} bundle exec rake release:announce
```

Comment:
Can we run this step when non tag push event too to detect such problems before we release a new version.
For example, generating release note step uses this approach.

We may be able to implement a "dry run" mode and use it for non tag push event.

Reply:
It sounds nice. I will implement it in this PR because we want to check whether it works or not.

Reply:
fix: [d890398](https://github.com/groonga/groonga/pull/2324/commits/d8903984cf5f1ed709b4bb3aa30e1a28b3deaa6c) I added the "dry run" mode and check the generated announcement on CI.

```
Run VERSION=${GITHUB_REF_NAME#v} bundle exec rake release:announce
  VERSION=${GITHUB_REF_NAME#v} bundle exec rake release:announce
  shell: /usr/bin/bash -e {0}
  env:
    DRY_RUN: true
    X_API_KEY: 
    X_API_KEY_SECRET: 
    X_ACCESS_TOKEN: 
    X_ACCESS_TOKEN_SECRET: 
git ls-files
Groonga 2324/merge (2025-05-08) has been released! This release adds the tokenizer's option to make token inspection simpler and improves negative-division semantics for unsigned integer. See: https://groonga.org/docs/news/15.html#release-2324/merge
```